addons/html_editor/static/src/core/history_plugin.js

```diff
@@ -917,10 +917,10 @@ export class HistoryPlugin extends Plugin {
      */
     unserializeNode(node) {
         let [unserializedNode, nodeMap] = this._unserializeNode(node);
-
-        for (const cb of this.resources["unserialize_node"] || []) {
-            unserializedNode = cb(unserializedNode);
-        }
+        const fakeNode = this.document.createElement("fake-el");
+        fakeNode.appendChild(unserializedNode);
+        this.shared.sanitize(fakeNode, { IN_PLACE: true });
+        unserializedNode = fakeNode.childNodes[0];
```

Comment:
While it does not seem wrong to put this logic here, the need to sanitize comes from the collaboration, not from the history plugin (if I have an editor without the collaboration plugins, I don't need to sanitize)

While pretty uncommon, unserializeNode could be called without the collaboration plugin i.e. if a node was inserted with the observer disconnected

Not really an issue for me though